modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java

```diff
@@ -819,6 +823,9 @@ private CPDefinition _upsertProduct(Product product) throws Exception {
 	@Reference
 	private CPOptionService _cpOptionService;
 
+	@Reference
+	private CProductLocalService _cProductLocalService;
```

Comment:
@andrea-ale-sbarra @Commando18 Why LocalService?

Reply:
We don't have a remote service for CProduct. Should we create a method in the CPDefinitionServiceImpl that calls CProductLocalService instead?

Reply:
yes please